apps/api/src/api/application-api.ts

```diff
@@ -116,3 +117,51 @@ export const getApplicationStateTotals = async ({ userId }: { userId: string })
 
 	return await service.applicationStateTotals({ user_id: userId });
 };
+/**
+ * Description placeholder
+ *
+ * @async
+ * @param {ApproveApplication} param0
+ * @param {ApproveApplication} param0.applicationId
+ * @returns {Promise<{
+* 	success: boolean;
+* 	message?: string;
+* 	errors?: string | Error;
+* 	data?: any;
+* }>}
+*/
+export const approveApplication = async ({ applicationId }: ApproveApplication): AsyncResult<ApplicationData[]> => {
+   try {
+	   // Fetch application
+	   const database = getDbInstance();
+	   const service: ApplicationService = applicationService(database);
+	   const result = await service.getApplicationById({ id: applicationId });
+
+	   if (!result.success) return result;
```

Comment:
`return result` should be wrapped with braces; team style convention
`if (!result.success) { return result; }`